conf/dbs.config

```
@@ -47,4 +47,12 @@ params {
         "alphafold2_multimer_v3" : "alphafold_params_colab_2022-12-06",
         "alphafold2_ptm"         : "alphafold_params_2021-07-14"
     ]
+
+    // Esmfold links
+    esmfold_3B_v1 = 'https://dl.fbaipublicfiles.com/fair-esm/models/esmfold_3B_v1.pt'
+    esm2_t36_3B_UR50D = 'https://dl.fbaipublicfiles.com/fair-esm/models/esm2_t36_3B_UR50D.pt'
+    esm2_t36_3B_UR50D_contact_regression = 'https://dl.fbaipublicfiles.com/fair-esm/regression/esm2_t36_3B_UR50D-contact-regression.pt'
```

it would probably look better like this 
```suggestion
    esmfold_3B_v1                        = 'https://dl.fbaipublicfiles.com/fair-esm/models/esmfold_3B_v1.pt'
    esm2_t36_3B_UR50D                    = 'https://dl.fbaipublicfiles.com/fair-esm/models/esm2_t36_3B_UR50D.pt'
    esm2_t36_3B_UR50D_contact_regression = 'https://dl.fbaipublicfiles.com/fair-esm/regression/esm2_t36_3B_UR50D-contact-regression.pt'
```